tools/reckless

```diff
@@ -637,8 +638,9 @@ def lightning_cli(*args, timeout=15) -> dict:
         raise CLIError(out['code'], out['message'])
     if clncli.returncode == 2:
         # RPC not available - lightningd not running or using alternate config
-        err = clncli.stderr.read().decode()
+        err = clncli.stderr.decode()
         raise RPCError(err)
+    raise Exception
```

Comment:
This is almost certainly wrong.... I would raise RPCError on any non-0/1 return?

Reply:
Sounds reasonable. Fixed!